js/views/conversation_view.js

```diff
@@ -341,16 +341,19 @@
       this.selectMember = this.selectMember.bind(this);
 
       const updateMemberList = async () => {
-        const maxToFetch = 1000;
-        const allMessages = await window.Signal.Data.getMessagesByConversation(
-          this.model.id,
-          {
-            limit: maxToFetch,
-            MessageCollection: Whisper.MessageCollection,
-          }
+        const allPubKeys = await window.Signal.Data.getPubkeysInPublicConversation(
+          this.model.id
         );
 
-        const allMembers = allMessages.models.map(d => d.propsForMessage);
+        const allMembers = await Promise.all(
+          allPubKeys.map(async pubKey => ({
+            id: pubKey,
+            authorPhoneNumber: pubKey,
+            authorProfileName: await ConversationController.get(
```

Comment:
I don't think it should be possible in this case but I have been burned before by ConversationController.get returning null so maybe we could use a combo of reduce and try/catch to be super safe?

Reply:
addressed